projects/storefrontapp-e2e-cypress/cypress/helpers/update-email.ts

```diff
@@ -54,10 +54,10 @@ export function updateEmail() {
   // TODO: uncomment below component and remove update-email assertion when #1957 is implemented
   cy.get('cx-update-email').should('exist');
   // checkBanner();
-  signOut();
 }
```

Comment:
I didn't mention it before, but I noticed when the test were joined using the standardUser breaks the test when it went from desktop to mobile, because it was the same data.

In line 12, we can remove standardUser that we use that generates a new email every time from requiredLoggedIn because they are separated test now and won't harm each other.

Reply:
However, there's a function to verify that the original 'standardUser' email is not valid anymore, because we generate a new email.